lib/src/base-styles/_nx-tile.scss

```diff
@@ -135,11 +135,19 @@ $nx-tile-background-color: $nx-white;
 }
 
 // On pages with tiles with multiple sections these are lighter weight titles
-.nx-tile__section-header {
-  border-top: 1px solid #b7bfdb;
-  margin-top: $nx-spacing-l;
-  margin-bottom: $nx-spacing-md;
-  padding-top: $nx-spacing-l;
+.nx-tile-subsection {
+  .nx-h3--tile-subsection {
+    margin-top: $nx-spacing-xl;
+    margin-bottom: $nx-spacing-md;
+  }
+
+  &:first-of-type {
+    .nx-h3--tile-subsection {
```

Comment:
Would it make more sense to put the border on the `<section>`?

Reply:
> Would it make more sense to put the border on the `<section>`?

I moved all the spacing and the border to the section: https://github.com/sonatype/sonatype-react-shared-components/pull/64/commits/0d21ac22dfb016139ac73ffb51c347c60e3e2eee